src/components/largePipelineWarning/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { toggleDisplayLargeGraph } from '../../actions/graph';
+import './largePipelineWarning.css';
+
+export const LargePipelineWarning = ({
```

Comment:
Also best to stick to the same naming style for all component directory names and filenames (kebab case), so here `large-pipeline-warning` throughout

Reply:
hhmm yea, I used pascal case out of second nature - I'll update the file name

Reply:
updated the component and file names